docker/mint/docker-run.sh

```diff
@@ -5,14 +5,18 @@
 set -ex
 export RUST_BACKTRACE=full
 
-seed=""
+params=""
 if [ -n "${CFG_SEED}" ]; then
-    seed="-s $CFG_SEED"
+    params="-s $CFG_SEED"
+fi
+
+if [ -n "${CFG_NUM_VALIDATORS}" ]; then # Total number of nodes in this network
+	  params+="-n ${CFG_NUM_VALIDATORS} "
 fi
 
 /opt/libra/bin/config-builder faucet \
     -o /opt/libra/etc \
-    $seed
+    $params \
```

Comment:
did you test this? I am not sure if this will work with the extra \

Reply:
I assume our build-docker test should catch this?

Reply:
No, it will not. This would require our docker build to actually run it. When it attempts to run it, it will crash loop.